src/app/public/modules/inline-form/types/inline-form-button.ts

```diff
@@ -0,0 +1,9 @@
+import {
+  SkyInlineFormButtonAction
+} from './inline-form-button-action';
+
+export interface SkyInlineFormButton {
```

Comment:
`SkyInlineFormButton` and `SkyInlineFormButtonConfig` are exactly the same other than the style being optional. Do we really need both types?

Reply:
Good eye. Theres a redundancy here. I've consolidated them to one.